CITATION.cff

```diff
@@ -0,0 +1,53 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: >-
+  MC particle transport simulations for the 2022
+  LET-measurements at DCPT
+message: >-
+  If you use this dataset, please cite it using the metadata
+  from this file.
+type: dataset
+authors:
+  - given-names: Niels
+    family-names: Bassler
+    affiliation: Aarhus Universitet
```

Comment:
"Aarhus University: Aarhus, DK"  to be consistent ?

Reply:
Done in [c9df4cb](https://github.com/APTG/2022_DCPT_LET/pull/70/commits/c9df4cb0ff07e2befc0593c3b8454b82bc02c738)